skyportal/handlers/api/comment.py

```diff
@@ -327,6 +343,7 @@ def post(self, associated_resource_type, resource_id):
                 bot=is_bot_request,
                 obj_id=spectrum.obj_id,
             )
+            obj_id = spectrum_id
```

Comment:
True. It was confusing as well so I removed it. It was used below in the code. Because I removed the line, I added this a few lines below  
```
elif associated_resource_type.lower() == "spectra":
            text_to_send = f"*@{self.associated_user_object.username}* mentioned you in a comment on *{spectrum_id}*"
            url_endpoint = f"/source/{spectrum_id}"
```